src/audio/PlaybackQueue.ts

```diff
@@ -149,6 +150,11 @@ export class PlaybackQueue {
                                 break; // Stop automatic playback: next useful event is not a voice message
                             }
 
+                            const isOwnVoiceMessage = cliID == event.sender.userId;
+                            if (isOwnVoiceMessage) {
```

Comment:
```suggestion
                            if (event.getSender() === MatrixClientPeg.get().getUserId()) {
```

I think this would be clearer

Reply:
I put the condition in the if statement but I am wondering if removing the const is worth the lost in performance as ```MatrixClientPeg.get().getUserId()`` would be called for each event and has always the same result.

Reply:
Good point!